samples/client/petstore/go-experimental/go-petstore/.openapi-generator/FILES

```diff
@@ -10,60 +10,60 @@ api_store.go
 api_user.go
 client.go
 configuration.go
-docs/AdditionalPropertiesAnyType.md
-docs/AdditionalPropertiesArray.md
-docs/AdditionalPropertiesBoolean.md
-docs/AdditionalPropertiesClass.md
-docs/AdditionalPropertiesInteger.md
-docs/AdditionalPropertiesNumber.md
-docs/AdditionalPropertiesObject.md
-docs/AdditionalPropertiesString.md
-docs/Animal.md
-docs/AnotherFakeApi.md
-docs/ApiResponse.md
-docs/ArrayOfArrayOfNumberOnly.md
-docs/ArrayOfNumberOnly.md
-docs/ArrayTest.md
-docs/BigCat.md
-docs/BigCatAllOf.md
-docs/Capitalization.md
-docs/Cat.md
-docs/CatAllOf.md
-docs/Category.md
-docs/ClassModel.md
-docs/Client.md
-docs/Dog.md
-docs/DogAllOf.md
-docs/EnumArrays.md
-docs/EnumClass.md
-docs/EnumTest.md
-docs/FakeApi.md
-docs/FakeClassnameTags123Api.md
-docs/File.md
-docs/FileSchemaTestClass.md
-docs/FormatTest.md
-docs/HasOnlyReadOnly.md
-docs/List.md
-docs/MapTest.md
-docs/MixedPropertiesAndAdditionalPropertiesClass.md
-docs/Model200Response.md
-docs/Name.md
-docs/NumberOnly.md
-docs/Order.md
-docs/OuterComposite.md
-docs/OuterEnum.md
-docs/Pet.md
-docs/PetApi.md
-docs/ReadOnlyFirst.md
-docs/Return.md
-docs/SpecialModelName.md
-docs/StoreApi.md
-docs/Tag.md
-docs/TypeHolderDefault.md
-docs/TypeHolderExample.md
-docs/User.md
-docs/UserApi.md
-docs/XmlItem.md
+docs\AdditionalPropertiesAnyType.md
+docs\AdditionalPropertiesArray.md
+docs\AdditionalPropertiesBoolean.md
+docs\AdditionalPropertiesClass.md
+docs\AdditionalPropertiesInteger.md
+docs\AdditionalPropertiesNumber.md
```

Comment:
the file path separator seems to differ between windows and linux, please revert this
cc @wing328

Reply:
Yeah I'm gonna make a whole different PR to fix this 🙁 

Some hard honest feedback: I must say, the experience of contributing to this PR has been a nightmare so far. I think more people would contribute if it was easier. This bug has existed for 5+ months, and I am sure many people have it - yet no one has contributed a fix yet. I suspect that's because it's very difficult.

The project is great though! 🙂 

So now, which parts do I remove, and which parts do I keep?

Reply:
@ffMathy sorry for hear that. We're in transition to a new process to update the samples. 

Please PM me via https://join.slack.com/t/openapi-generator/shared_invite/enQtNzAyNDMyOTU0OTE1LTY5ZDBiNDI5NzI5ZjQ1Y2E5OWVjMjZkYzY1ZGM2MWQ4YWFjMzcyNDY5MGI4NjQxNDBiMTlmZTc5NjY2ZTQ5MGM

Reply:
Sorry, I can't contribute much to anything else than solving this issue, since I am a consultant and I am being paid to work for a company. They allowed me to fix the bug that causes them to be stuck, but not more than that.

I hope you understand. Can you let me know when the Windows build script has been fixed, so I can regenerate using that?

Reply:
I just updated the samples via https://github.com/OpenAPITools/openapi-generator/pull/6538/commits/5a754e314c2c5398172f01304399a2bef85dc7cc. Let's see how it goes.